etcd/getting-started-with-etcd.md

```diff
@@ -6,6 +6,36 @@ Application containers running on your cluster can read and write data into etcd
 
 <a class="btn btn-default" href="../latest/api.html">Complete etcd API Docs</a>
 
+## Setting up etcd
+
+Container Linux comes pre-installed with an `etcd-wrapper` script which can be configured to **run etcd from inside of a container**. Container Linux also comes pre-installed with an `etcd-member.service` systemd unit which by default uses the etcd3 image. This means you to use etcd3 without needing to install the etcd3 binary.
+
+To enable the `etcd-member.service` unit you need to manually disable the `etcd2.service` unit and enable the `etcd-member.service`.
```

Comment:
@philips On my vagrant install of CL alpha (1325.0.0) etcd2.service is enabled by default. On other installs it may be different. I'll word this to reflect the "Do this **if** etcd2.service is enabled."